### PR TITLE
[Refactor] Merge ti.tools.image.imdisplay() into ti.tools.image.imshow()

### PR DESCRIPTION
**Pull request by neozhaoliang**
Note `Ipython.display` can directly show `Image.fromarray()`, no need to write it into BytesIO and read it back.

### CODE REVIEW

**Comment by strongoier** [comment]
Would you resolve conflicts and try to make CI happy? Thanks!

**Comment by neozhaoliang** [comment]
Oh, I forgot to remove the `import imdisplay` in `tools.__init__.py`, should be fixed now.

File: python/taichi/tools/image.py [hunk 1]
```diff
@@ -113,26 +91,36 @@ def imread(filename, channels=0):
...
+                gui.set_image(img)
+                gui.show()
+    else:
+        import IPython.display  # pylint: disable=C0415
+        import PIL.Image  # pylint: disable=C0415
+        img = cook_image_to_bytes(img)
+        IPython.display.display(PIL.Image.fromarray(img))
 
 
 __all__ = ['imdisplay', 'imread', 'imresize', 'imshow', 'imwrite']
```
Comment (strongoier) [inline python/taichi/tools/image.py:126]:
```suggestion
__all__ = ['imread', 'imresize', 'imshow', 'imwrite']
```

**Comment by strongoier** [comment]
You also have to sign the CLA :-)

**Review by strongoier — APPROVED** [review]
LGTM!

**Pull request merged**